src/components/D3Rings.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <div />
+</template>
+<script>
+import mapboxgl from "mapbox-gl";
+import * as d3 from "d3";
+import tour from "../assets/monitoring_locations/delawareBasinNextGenerationLocationsSorted";
+export default {
+  name: "D3Rings",
+  mounted() {
+    this.$root.$on("CreateRings", () => {
+      this.CreateRings();
+    });
+    this.$root.$on("RemoveRings", () => {
+      this.RemoveRings();
+    });
+  },
+  methods: {
+    CreateRings() {
+      let map = this.$store.map;
+      let $this = this;
+      let radius = 15;
+      //Gage array entries match up to the index of the color array
+      let gageArray = ["enhanced_gage", "new_gage", "specific_conductance", "temperature", "camera"];
+      let colors = ["#00F4CE", "#2B9BFD", "#FEBD03", "#FE324D", "#FF562F"];
```

Comment:
Wondering if we can make this dynamic by adding the color to the JSON as part of that boolean, like 'cameras: [true, '#00F4CE']'. Yeah, I think that would work as issue too. So, let's go with this as is, and make this dynamic in a new issue.